src/pygambit/game.pxi

```diff
@@ -672,16 +672,20 @@ class Game:
                     mspr[s] = Rational(v)
             return mspr
 
-    def mixed_behavior_profile(self, rational=False) -> MixedBehaviorProfile:
-        """Create a behavior strategy profile over the game.
-
-        The profile is initialized to uniform randomization for each player
-        over their actions at each information set.
+    def mixed_behavior_profile(self, data=None, rational=False) -> MixedBehaviorProfile:
+        """Returns a behavior strategy profile `MixedBehaviorProfile` over the game,
+        initialized to uniform randomization for each player over his actions at each
+        information set. If ``data`` is not specified, the mixed strategy profile is 
+	initialized to uniform randomization for each player over his actions.
 
         Parameters
         ----------
-        rational
-            If True, probabilities are represented using rational numbers; otherwise
+	data
+            A nested list (or compatible type) with the same dimension as the action set of the game,
+            specifying the probabilities of the strategies.
+        
+	rational
+            If `True`, probabilities are represented using rational numbers; otherwise
```

Comment:
It appears that numpydoc convention is that when referring to Python constants like True, False, None and so on, not to enclose them in any back-ticks at all (as in the original version of the docstring)